### PR TITLE
Changed and fixed some typos/grammar.

### PR DESCRIPTION
**Pull request by sameetmehta**
There are a couple of places where either the code itself is not explained properly, e.g. why use `tasklogger`.  If the aim is to show the difference in efficiencies, then that must be said so explicitly.  I would rather that it be said explicitly here, than mentioning it in the lectures.  The notebooks should ideally be as complete as possible in themselves.

### CODE REVIEW

File: exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb [hunk 1]
```diff
@@ -14,7 +14,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We'll install `louvain` by downloading a precompiled version, because it takes a while. You could install it simply by running\n",
+    "We'll install `louvain` by downloading a precompiled version, because it takes a while. You could install it simply by running.  `louvain` is a community detection algorithm.\n",
```
Comment (sameetmehta) [inline exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb]: A bit more clarification for `louvain`.

File: exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb [hunk 2]
```diff
@@ -754,7 +761,7 @@
     "\n",
     "1. Use `scprep.plot.histogram` (https://scprep.readthedocs.io/en/dev/reference.html#scprep.plot.histogram). \n",
     " * Note, you can pass an list of expression values to get them plotted on the same axis.\n",
-    " * Try using `log='y'` to get a log-scale y-axis\n",
+    " * Try using `log='y'` to get a log-scale y-axis.  This will typically plot the values at the natural log scale.  If you want to different scale, you may have to do one or two additional steps.  **Note Dan/Scott**: Any tips on plotting on log2 scale or log10 for that matter.\n",
```
Comment (scottgigante) [inline exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb]: Using `log='y'` doesn't change the actual values, only the scaling of them, so there is actually no 'base' to this log. See the following as an example.
```
import numpy as np
import matplotlib.pyplot as plt

x = np.arange(100)
y = -(x-50)**2 + 50**2 + 1

plt.figure()
plt.plot(x, y)
plt.yscale('log')
plt.title('log scale')

plt.figure()
plt.plot(x, np.log(y))
plt.title('log(y)')

plt.figure()
plt.plot(x, np.log2(y))
plt.title('log2(y)')

plt.figure()
plt.plot(x, np.log10(y))
plt.title('log10(y)')

plt.show()
```

File: exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb [hunk 3]
```diff
@@ -693,7 +700,7 @@
     "\n",
     "In the above exercise, we manually inspected known marker genes to annotate clusters. Here, we're going to simply identify the genes that change the most between two groups. As discussed in lecture, there are several ways to calculate differential expression. We are going to focus on the mean-difference and the rank-sum statistic.\n",
     "\n",
-    "You can easily calculate basic differential expression statistics using `scprep.stats.differential_expression` and `scprep.stats.differential_expression_by_cluster`. Documentation for these methods can be found at: https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression\n",
+    "You can easily calculate basic differential expression statistics using `scprep.stats.differential_expression` and `scprep.stats.differential_expression_by_cluster`. Documentation for these methods can be found at: https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression.\n",
```
Comment (scottgigante) [inline exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb]: Adding a period at the end of the URL is risky. Can we fix this with markdown? 
```
[https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression](https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression).
```
renders as [https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression](https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression).

**Review by scottgigante — CHANGES_REQUESTED** [review]

File: exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb (hunk 2 above)
Comment (sameetmehta) [reply exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb]: I will make the change.  I think the question you asked needs to go in the notebook to clarify why we are scaling the y-axis -- the dynamic range is too big.

File: exercises/Clustering/notebooks/02_Clustering_retinal_bipolar.ipynb [hunk 4]
```diff
@@ -772,36 +780,18 @@
...
-       "  'Kcn4',\n",
-       "  'Cabp5',\n",
-       "  'Vsx1',\n",
-       "  'Prkca']}"
-      ]
-     },
-     "execution_count": 43,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "SyntaxError",
```
Comment (scottgigante) [inline exercises/Clustering/notebooks/02_Clustering_retinal_bipolar.ipynb:789]: Looks like we have an error here?

File: exercises/Clustering/notebooks/02_Clustering_retinal_bipolar.ipynb [hunk 5]
```diff
@@ -811,123 +801,11 @@
...
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "source": [
+    "**Note Dan/Scott**: Is this is intended endpoint of this notebook.  Is there supposed to be more stuff happening. Feels a bit incomplete."
```
Comment (scottgigante) [inline exercises/Clustering/notebooks/02_Clustering_retinal_bipolar.ipynb:807]: Looks like this notebook is not included in the current [workshop schedule](https://www.krishnaswamylab.org/workshop). Looks like we didn't end up using it.

File: exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb (hunk 2 above)
Comment (scottgigante) [reply exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb]: This is marked as resolved but does not appear to be resolved.

Comment (sameetmehta) [reply exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb]: Should have been.  Let me check again.

**Review by scottgigante — APPROVED** [review]

**Pull request merged**